applications/StructuralMechanicsApplication/custom_response_functions/response_utilities/adjoint_linear_strain_energy_response_function.cpp

```diff
@@ -56,29 +57,9 @@ namespace Kratos
     {
         KRATOS_TRY;
 
-        ProcessInfo &r_current_process_info = rModelPart.GetProcessInfo();
-        double response_value = 0.0;
+        LinearStrainEnergyResponseFunction zero_order_response(rModelPart, mResponseSettings);
```

Comment:
@msandre This is what we discussed: When the response value is requested at a "adjoint" response function, the much simpler utility to just calculate the value is used.